models/user_test.go

```diff
@@ -0,0 +1,133 @@
+package models_test
+
+import (
+	_ "google-scraper/initializers"
+	. "google-scraper/tests/fabricators"
+	. "google-scraper/tests/testing_helpers"
+	"google-scraper/models"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("User", func() {
+	Describe("#CreateUser", func() {
+		Context("given the user with valid params", func() {
+			It("returns the user ID", func() {
+				user := &models.User{
+					Name:           "John",
+					Email:          "john@example.com",
+					HashedPassword: "secret",
+				}
+				userId, err := models.CreateUser(user)
+				if err != nil {
+					Fail("Adding user failed: " + err.Error())
+				}
+
+				Expect(userId).ToNot(BeNil())
+			})
+
+			It("returns empty error", func() {
+				user := &models.User{
+					Name:           "John",
+					Email:          "john@example.com",
+					HashedPassword: "secret",
+				}
+				_, err := models.CreateUser(user)
+
+				Expect(err).To(BeNil())
+			})
+		})
+
+		Context("given the user with INVALID params", func() {
```

Comment:
@junan How about adding cases when the params are missing?

Reply:
@carryall As per our discussion, we are ignoring it as I don't have validation for other columns